### PR TITLE
Rename DefaultPolicies to ConiksPolicies

### PR DESCRIPTION
**Pull request by c633**
As discussed in #6

### CODE REVIEW

File: merkletree/policy.go [hunk 1]
```diff
@@ -14,23 +14,23 @@ type Policies interface {
 	vrfPrivate() *[vrf.SecretKeySize]byte
 }
 
-type DefaultPolicies struct {
+type ConiskPolicies struct {
```
Comment (masomel) [inline merkletree/policy.go]: typo: ``Conisk`` should be ``Coniks``.

Comment (c633) [inline merkletree/policy.go]: Oops :(

Comment (masomel) [inline merkletree/policy.go]: No problem!

**Comment by masomel** [comment]
I think this is ready to be merged. The additional mandatory policy fields will be added as part of #35 (as we decided in https://github.com/coniks-sys/coniks-go/pull/37#issuecomment-235768526), right?

**Comment by c633** [comment]
> The additional mandatory policy fields will be added as part of #35 (as we decided in #37 (comment)), right?

Right.

**Pull request merged**